client/src/components/Features/Features.tsx

```diff
@@ -147,28 +132,20 @@ const Features: React.FC<Props> = () => {
                   <BulletIcon color="primary" />
                 </ListItemIcon>
                 <Typography color="textSecondary">
-                  Bridge the gap between a high-level visual understanding of an
-                  algorithm and the implementation itself
+                  Bridge the gap between a high-level visual understanding of an algorithm and the
+                  implementation itself
                 </Typography>
               </ListItem>
             </List>
           </Grid>
-          <Grid
-            item
-            xs={12}
-            sm={12}
-            md={4}
-            sx={{ position: 'relative', height: '200px' }}
-          >
+          <Grid item xs={12} sm={12} md={4} sx={{ position: 'relative', height: '200px' }}>
             <LaptopFrame imageUrl={visualiserDashboardLessonScreen} />
           </Grid>
         </Grid>
       </FeatureItem>
       <FeatureItem fromDirection="left">
         <Typography color="textPrimary" variant="h6" sx={{ textAlign: 'left' }}>
-          <EditIcon />
-          {' '}
-          Content Management
+          <EditIcon /> Content Management
```

Comment:
We should remove these descriptions on the homepage for now because we don't have them on the site yet.

Reply:
Good catch. Removed.